lib/kg/accounts/user.ex

```diff
@@ -26,15 +28,19 @@ defmodule Kg.Accounts.User do
   """
   def registration_changeset(user, attrs) do
     user
+<<<<<<< HEAD
+    |> cast(attrs, [:email, :password, :first_name, :last_name])
+=======
     |> cast(attrs, [:email, :password, :address])
+>>>>>>> 0a9e7ac537cee73466e35cd7bda4950ee7e2a793
     |> validate_email()
     |> validate_password()
   end
 
   defp validate_email(changeset) do
     changeset
     |> validate_required([:email])
-    |> validate_format(:email, ~r/^[^\s]+@[^\s]+$/, message: "must have the @ sign and no spaces")
+    |> validate_format(:email, ~r/^[^\s]+@[^\s]+\.+[^\s]+$/, message: "Please enter a valid email")
```

Comment:
Are we overwriting an old change by accident or is this what you want?

Reply:
That was intended. Bad idea? I noticed that a user could sign up using myemail@nodomain so I thought that would help.

Reply:
nope I misread the code, that sounds good 👍 

This error message gets passed down into the view, so some of the tests now fail. I'll talk to you in iMessage to explain how to run the tests.